autoendpoint/src/error.rs

```diff
@@ -195,9 +194,9 @@ impl ApiErrorKind {
             ApiErrorKind::General(_) => "general",
             ApiErrorKind::Io(_) => "io",
             ApiErrorKind::Metrics(_) => "metrics",
-            ApiErrorKind::Database(_) => "database",
+            ApiErrorKind::Database(e) => e.metric_label().unwrap_or("database"),
```

Comment:
Should probably just return on these?
```suggestion
            ApiErrorKind::Database(e) => return e.metric_label(),
```

Reply:
I'll also fix up the other one too.